backend/rust/bunker/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+/// Import the code() backend.rs was generated from backend.proto
+pub mod pb {
+    include!("../generated/backend.rs");
+}
+
+pub mod service;
```

Comment:
```suggestion
use tonic::transport::Server;

pub use crate::pb::backend_server::Backend as BackendService;
use crate::pb::backend_server::BackendServer;

// Run the backend with the default behavior
pub async fn run(backend: impl BackendService, addr: impl Into<SocketAddr>) -> anyhow::Result<()> {
    let svc = BackendServer::new(backend);

    let r = Server::builder()
        .add_service(svc)
        .serve(addr.into())
        .await?;

    Ok(r)
```
Something like this.